### DISCUSSION TITLE
Compass Watch

### DISCUSSION THREAD

**Discussion by hanixshah** [CubicRP/Cubic#17] [Ideas]
Suggestion Title: Add Compass Watch to digital store and LSPD Armory

Description: 

I really like the hud we have in the server tight now and also the compass on the top but if someone needs to callout directions it's pretty hard, such as LSPD pursuits comms and stuff like that. So once someone use/turns on the compass watch it should be turned on.
It will help with passing over good comms and exact location with landmarks if needed.

Example: https://imgur.com/yZYWQvq

**Comment by Gibuso** [comment]
im guessing u meant to say to add street names next to the compass.

I kinda like this. 
![image](https://user-images.githubusercontent.com/100157207/185269841-4938533d-37da-4546-8f90-ec278fd9de09.png)

**Comment by dwreyyes** [comment]
I brought this up when we put this HUD in, so it didn't go unnoticed. I think it will be important for PD as well as Park Rangers, and EMS.
